api/client/proto/authservice.proto

```diff
@@ -841,6 +841,9 @@ message GetEventsRequest {
     // If the previous response had LastKey set then this should be
     // set to its value. Otherwise leave empty.
     string StartKey = 6;
+    // Order specifies an ascending or descending order of events.
+    // A value of 0 means a descending order and a value of 1 means an ascending order.
+    int32 Order = 7;
```

Comment:
Nit: use a GRPC enum with descending as the zero value.  Same effect, but results in much clearer proto definition. Ex:

```protobuf
enum Order {
    DESCENDING = 0;
    ASCENDING = 1;
}
```

Reply:
Done